audience-analytics/index.rst

```diff
@@ -0,0 +1,289 @@
+.. _audience-tuto:
+
+Audience Analytics
+==================
+
+|
+
+Step 1 - Introduction
+*********************
+
+This documentation is meant at developers wanting to install, configure and launch Angus.ai audience analytics application on a personal computer.
+
+What data can be retrieved
+--------------------------
+
+.. image:: ../images/intro.png
+
+Angus.ai anonymous audience analytics solution computes (from each video stream) the following metrics:
+
+- The number of people passing by the camera/device,
+- The number of people interested in the camera/device
+- The time spent stopped in front of the camera/device
+- The time spent looking at the camera/device
+- The number of people interested by the camera/device, broken down by
+  demographics
+
+  - Age
+  - Gender
+  - Emotion
+
+For more information about the metrics, see the page dedicated to :ref:`the metrics <metrics>`.
+
+How it works
+------------
+
+Angus.ai audience analytics solution is based on a (lightweight) Client / Server architecture as seen on the figure below.
+All CPU expensive computation are made on our dedicated servers making it possible to run the solution from about any CPU board that can retrieve a camera stream and connect to a server (eg. Raspberry).
+
+.. image:: ../images/archi.jpeg
+
+Once properly installed and configured, this application will interact with Angus.ai cloud based algorithms to provide audience metrics that can be retrieve through a REST API.
+This tutorial will show how to do it.
+
+Requirements
+------------
+
+As you go through this tutorial, you will need:
+
+- a computer. Every operating system is ok provided that you can configure a Python or Java stack.
+- a camera (e.g. webcam) plugged into that computer. USB and IP cameras are supported, although IP cam can be more challenging to interface. If you need help doing so please contact us at support@angus.ai.
+- a working internet connection. An upload bandwidth of about 400ko/sec is advised. If this is a problem, we are able to provide an "hybrid" version of our solution, where part of the CPU expensive computation is done locally, alleviating connection bandwidth requirements. Please contact us at support@angus.ai.
```

Comment:
You can reduce this bandwidth by set a different rate compression before sending image to the cloud.